packages/vast-xml2js/src/helpers/parseFromString.js

```diff
@@ -0,0 +1,13 @@
+import saneError from 'sane-domparser-error';
+
+const parseFromString = (DOMParser, xmlText) => {
```

Comment:
What is the benefit of doing dependency injection for DOMParse here? Is there a case when a `DOMParser` will be different?

Reply:
There are two builds one for the browser and one for node. Dependency injection here is just to be able to chose which DOMParser I should use